### ISSUE TITLE
Preserve plugin status on plugin action links

### ISSUE THREAD

**Issue by sanzeeb3** [audrasjb/wp-autoupdates#49]
Similar to #46, the plugin status should also be preserved on plugin 'Activate', 'Deactivate', 'Delete', etc. Currently, for these plugin actions, the users are redirected to 'All'.

**Comment by pbiron** [comment]
I'm not seeing the behavior you describe.

This plugin does not make any modifications to the row actions (Activate, Deactivate, etc) and core correctly supplies the `plugin_status` query arg on those links.

Is it possible that you've got another plugin installed that is modifying the URLs for those row action links?

**Comment by sanzeeb3** [comment]
I don't have any other plugins modifying URLs. To reproduce the issue, go to  "Auto-updates Disabled" and deactivate any plugin, you'll be redirected back to "All".

**Comment by pbiron** [comment]
> I don't have any other plugins modifying URLs. To reproduce the issue, go to "Auto-updates Disabled" and deactivate any plugin, you'll be redirected back to "All".

Thanx for the context (pun intended...you'll get the joke if you look at https://core.trac.wordpress.org/browser/trunk/src/wp-admin/includes/class-wp-plugins-list-table.php#L621).

The incorrect behavior in that case is what I consider a bug in `WP_Plugins_List_Table`.  It only happens when the current "view" is a custom view (such as "Auto-updates Disabled").  I've encountered this before when adding custom views to this list table but I forgot all about it.

`WP_Plugins_List_Table::__construct()` sets a global variable for the current "view".  If the current view is a custom view, then it incorrectly (IMHO) sets that global variable to 'all'...which I think is a bug.  I'll open a Trac ticket about that.

And now I see that I even added a comment about this when I did the PR that added the custom views (https://github.com/audrasjb/wp-autoupdates/blob/master/wp-autoupdates.php#L366)...and forgot about that as well :-)

When this plugin gets merged into core, the 2 custom "views/statuses" that this plugin adds will be added to the reserved list in `WP_Plugins_List_Table::__construct()`.  Therefore, I propose we close this issue and address it in the Trac in the ticket that I will create.  @sanzeeb3, are you OK with that?
